src/parachain/escrow.ts

```diff
@@ -134,12 +145,29 @@ export class DefaultEscrowAPI implements EscrowAPI {
         await this.transactionAPI.sendLogged(tx, this.api.events.escrow.Deposit, true);
     }
 
+    async getRewards<U extends GovernanceUnit>(accountId: AccountId): Promise<MonetaryAmount<Currency<U>, U>> {
+        // Step 1. Get amount in reward pool for the account ID
+        const [rewardStake, rewardPerToken, rewardTally] = await Promise.all([
+            this.getEscrowStake(accountId),
+            this.getRewardPerToken(),
+            this.getRewardTally(accountId)
+        ]);
+        // Step 2. Calculate the rewards that can be withdrawn at the moment
+        // Stake[currencyId, accountId] * RewardPerToken[currencyId] - RewardTally[currencyId, accountId]
+        const rewards = rewardStake.mul(rewardPerToken).sub(rewardTally);
```

Comment:
@nud3l we already have a function for this in the lib, would be good to reuse it
https://github.com/interlay/interbtc-api/blob/09f7be92435eb6c16f5e7184b0bd016afb5c451c/src/utils/currency.ts#L44